src/components/Navbar.jsx

```diff
@@ -1,27 +1,29 @@
 import { useNavigate } from "react-router-dom";
```

Comment:
- [ ] Please as the last reviewer said you should work on your code and make it match the design currently these are the things that are missing in your design : 
- the navigation it doen't match the design the background should be white and the height should look as in the design  
- the chapter name and the button should match the design , your code 
![image](https://github.com/kimitaWanjohi/bookstore/assets/74744787/5b5d1759-f1d4-45fb-a0f2-ad7f22ea398e)
 the design 
![image](https://github.com/kimitaWanjohi/bookstore/assets/74744787/533af3fa-4e11-4498-ad97-bdf5987a910c)
- there is a line between the chapter info and the progress bar ]
- the distance between the chapter  info and the progress bar should match the design 
![image](https://github.com/kimitaWanjohi/bookstore/assets/74744787/6633502b-f2e6-4922-9ba2-f4d5b52e98b2)
- the border of the input field shoudl match the design and the text inside it color should match the design 
![image](https://github.com/kimitaWanjohi/bookstore/assets/74744787/bcf89010-7aec-41be-b997-edf76c057a98)
- the background of the card section should match the design 
![image](https://github.com/kimitaWanjohi/bookstore/assets/74744787/45220fad-770b-4f1b-874d-e841d6e99b2b)

- the distance between each book card should match the design 
![image](https://github.com/kimitaWanjohi/bookstore/assets/74744787/84928757-2278-48bf-8340-9d06a935812d)